src/components/views/rooms/RoomSublist2.tsx

```diff
@@ -71,6 +71,8 @@ interface IProps {
     // You should feel bad if you use this.
     extraBadTilesThatShouldntExist?: React.ReactElement[];
 
+    onResize();
+
```

Comment:
should cram this up above under the `tagId` as it's just a general option. Also would prefer to match the code style of the add room button.